entries/b/baloise.ch.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Baloise Insurance": {
+    "domain": "baloise.ch",
+    "img": "baloise.ch.png",
```

Comment:
This is the logo in SVG format, which we prefer:
![baloise ch](https://user-images.githubusercontent.com/520841/181715891-53c5eb83-ea6f-4197-92c3-6593980c8fc6.svg)